app/models/user.rb

```diff
@@ -1,9 +1,13 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  #  :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] You did a great job adding `devise` to the users model and it works very well, but as part of this project's requirements, users are required to confirm their emails before they access the app.

![email](https://user-images.githubusercontent.com/53124548/225594871-5f977e6a-0b52-4f77-8e4e-8dd38fb9dd4a.png)